### PR TITLE
try to clean grpc for millionth time

### PR DESCRIPTION
**Pull request by krsna1729**
Signed-off-by: Saikrishna Edupuganti <saikrishna.edupuganti@intel.com>

### CODE REVIEW

**Comment by ajamshed** [comment]
@krsna1729 ,

I see the following error:

```
Step 45/48 : RUN make $MAKEFLAGS &&     cp zmq-cpiface /bin
 ---> Running in b1c795f9860f
g++ -g -O3 -std=c++17 -Werror -Wall -Wcast-align *.cc -I./pb ./pb/*.pb.o -o zmq-cpiface -l:libprotobuf.a -l:libgrpc++.a -l:libgrpc_unsecure.a -l:libcares.a -lpthread -lz -lzmq -lglog
/tmp/ccwheUZa.o: In function `std::_Sp_counted_ptr<decltype(nullptr), (__gnu_cxx::_Lock_policy)2>::_M_dispose()':
/cpiface/bess_control.h:42: multiple definition of `sig_handler(int)'
/tmp/ccsVMaGj.o:/cpiface/zmq-cpiface-central-cp-multi-upfs.cc:139: first defined here
/tmp/ccwheUZa.o:/cpiface/zmq-cpiface.cc:28: multiple definition of `receiver'
/tmp/ccsVMaGj.o:/cpiface/zmq-cpiface-central-cp-multi-upfs.cc:28: first defined here
/tmp/ccwheUZa.o:/cpiface/zmq-cpiface.cc:29: multiple definition of `sender'
/tmp/ccsVMaGj.o:/cpiface/zmq-cpiface-central-cp-multi-upfs.cc:29: first defined here
/tmp/ccwheUZa.o:/cpiface/zmq-cpiface.cc:32: multiple definition of `context1'
/tmp/ccsVMaGj.o:/cpiface/zmq-cpiface-central-cp-multi-upfs.cc:32: first defined here
/tmp/ccwheUZa.o:/cpiface/zmq-cpiface.cc:33: multiple definition of `context2'
/tmp/ccsVMaGj.o:/cpiface/zmq-cpiface-central-cp-multi-upfs.cc:33: first defined here
/tmp/ccwheUZa.o: In function `operator<<(std::ostream&, ip_addr const&)':
/cpiface/bess_control.h:46: multiple definition of `operator<<(std::ostream&, ip_addr const&)'
/tmp/ccsVMaGj.o:/cpiface/bess_control.h:96: first defined here
/tmp/ccwheUZa.o: In function `BessClient::runRemoveCommand(unsigned int, char const*)':
/usr/include/c++/7/bits/locale_facets.h:874: multiple definition of `main'
/tmp/ccsVMaGj.o:/cpiface/zmq-cpiface-central-cp-multi-upfs.cc:148: first defined here
/tmp/ccwheUZa.o:/cpiface/zmq-cpiface.cc:31: multiple definition of `context0'
/tmp/ccsVMaGj.o:/cpiface/zmq-cpiface-central-cp-multi-upfs.cc:31: first defined here
/tmp/ccwheUZa.o:/cpiface/zmq-cpiface.cc:30: multiple definition of `reg'
/tmp/ccsVMaGj.o:/cpiface/zmq-cpiface-central-cp-multi-upfs.cc:30: first defined here
collect2: error: ld returned 1 exit status
make: *** [all] Error 1
Makefile:11: recipe for target 'all' failed
```

**Comment by krsna1729** [comment]
@ajamshed seems like you have copy of cpiface file in same directory causing compilation issue?

File: cpiface/Makefile [hunk 1]
```diff
@@ -4,26 +4,11 @@
...
-	$(PROTOC) -I$(PBDIR) -I$(PROTOBUF_INC) --cpp_out=. $(PBDIR)/*.proto
-	$(PROTOC) -I$(PBDIR) -I$(PROTOBUF_INC) --grpc_out=. --plugin=protoc-gen-grpc=`which grpc_cpp_plugin` $(PBDIR)/*.proto
-
-c-objs: proto-stuff
-	$(CXX) $(CXXFLAGS) -c -o zmq-cpiface.o zmq-cpiface.cc -I$(PBDIR) -I$(GRPC_INC) -I$(PROTOBUF_INC)
-
-proto-objs: proto-stuff
-	$(CXX) $(CXXFLAGS) *.pb.cc -c -I./ -I$(PBDIR) -I$(PROTOBUF_INC) -I$(GRPC_INC)
+all:
+	$(CXX) $(CXXFLAGS) *.cc -I$(PBDIR) $(PBDIR)/*.pb.o -o $(TARGET) -l:libprotobuf.a $(GRPC_LIBS) -lpthread -lz -lzmq -lglog
```
Comment (ajamshed) [inline cpiface/Makefile:11]:
```suggestion
	$(CXX) $(CXXFLAGS) $(TARGET).cc -I$(PBDIR) $(PBDIR)/*.pb.o -o $(TARGET) -l:libprotobuf.a $(GRPC_LIBS) -lpthread -lz -lzmq -lglog
```

Comment (ajamshed) [reply cpiface/Makefile:11]: This was causing the multiple definition errors. If only the target source file could be compiled, then that would be great. The rest LGTM.

Comment (krsna1729) [reply cpiface/Makefile:11]: That would not work if someone would override target variable.

Comment (krsna1729) [reply cpiface/Makefile:11]: Merging as is. We can add sources variable later.

**Pull request merged**